libsplinter/src/rest_api/auth/actix.rs

```diff
@@ -184,6 +184,16 @@ where
                     .into_future(),
                 )
             }
+            AuthorizationResult::UnknownEndpoint => {
```

Comment:
Does this mean it it is possible to add an endpoint that would be unknown to authorization?

Reply:
No; all endpoints require a permission field, so you can't add an endpoint that authorization wouldn't know about. If authorization doesn't know about an endpoint, the endpoint is unknown to the whole REST API.

Reply:
Does that mean this is unreachable?

Reply:
No, this is what enables us to detect 404's. The middleware is run before Actix matches the request to a resource definition, so we have to distinguish between a known endpoint where the client is authorized and an unknown endpoint (which we can't determine how to authorize).